File: 2427. Number of Common Factors.java
class Solution {
    public int commonFactors(int a, int b) {
        int max = Math.max(a, b);
        int min = Math.min(a, b);
        int res = 0;

        for(int i = 1; i<=min; i++){
            if(max%i == 0 && min%i == 0) res++
        }

        return res;
    }
}

File: 2427. Number of Common Factors.ts
function commonFactors(a: number, b: number): number {
    let max = Math.max(a,b);
    let min = Math.min(a,b);
    let res = 0;

    for(let i = 0; i <= min; i++){
        if(min % i == 0 && max % i == 0) res++;
    }

    return res;
};
